Makefile

```diff
@@ -1,24 +1,27 @@
 PROJECT = rabbitmq_email
 PROJECT_DESCRIPTION = RabbitMQ plugin that converts incoming emails into messages and messages into outgoing emails
+RABBITMQ_VERSION = v3.9.x
```

Comment:
```
RABBITMQ_VERSION ?= v3.9.x
```

Reply:
That said, I never remember how the variables on the command line affect the Makefile, so it might not be strictly necessary.